### DISCUSSION TITLE
Restructure the Documentation into category

### DISCUSSION THREAD

**Discussion by farodin91** [JanusGraph/janusgraph#2336] [Ideas]
I started to restructure the documentation into category:
Table of Content:
  - Introduction: index.md
  - Getting Started:
    - Installation: getting-started/installation.md
    - Basic Usage: getting-started/basic-usage.md
    - Architectural Overview: getting-started/architecture.md
  - Configuration:
    - Introduction: configs/index.md
    - Configuration Reference: configs/configuration-reference.md
    - Config Example: configs/example-config.md
  - Schema Management:
    - Schema and Data Modeling: schema/index.md
    - Advanced Schema: schema/advschema.md
    - Index Management:
      - Indexing for Better Performance: schema/index-management/index-performance.md
      - Index Lifecycle: schema/index-management/index-lifecycle.md
      - Reindexing: schema/index-management/index-reindexing.md
      - Removal: schema/index-management/index-removal.md
  - Operations:
    - JanusGraph Server: operations/server.md
    - Deployment Scenarios: operations/deployment.md
    - Dynamic Graphs: operations/dynamic-graphs.md
    - ConfiguredGraphFactory: operations/configured-graph-factory.md
    - Bulk Loading: operations/bulk-loading.md
    - Monitoring: operations/monitoring.md
    - Migrating from Titan: operations/migrating-titan.md
    - Migrating from Thrift: operations/migrating-thrift.md
  - Storage Backends:
    - Introduction: storage-backend/index.md
    - Apache Cassandra: storage-backend/cassandra.md
    - Apache HBase: storage-backend/hbase.md
    - Google Cloud Bigtable: storage-backend/bigtable.md
    - Oracle Berkeley DB Java Edition: storage-backend/bdb.md
    - InMemory Storage Backend: storage-backend/inmemorybackend.md
  - Mixed Index Backends:
    - Introduction: index-backend/index.md
    - Index Parameters and Full-Text Search: index-backend/text-search.md
    - Field Mapping: index-backend/field-mapping.md
    - Direct Index Query: index-backend/direct-index-query.md
    - Elasticsearch: index-backend/elasticsearch.md
    - Apache Solr: index-backend/solr.md
    - Apache Lucene: index-backend/lucene.md
  - Interactions with JanusGraph:
    - Gremlin Query Language: interactions/gremlin.md
    - Transactions: interactions/transactions.md
    - Search Predicates and Data Types: interactions/search-predicates.md
    - Connecting to JanusGraph:
      - Introduction: interactions/connecting/index.md
      - Using Java: interactions/connecting/java.md
      - Using Python: interactions/connecting/python.md
      - Using .Net: interactions/connecting/dotnet.md
  - Advanced Topics:
    - Eventually-Consistent Storage Backends: advanced-topics/eventual-consistency.md
    - Failure & Recovery: advanced-topics/recovery.md
    - Graph Partitioning: advanced-topics/partitioning.md
    - Datatype and Attribute Serializer Configuration: advanced-topics/serializer.md
    - JanusGraph with TinkerPop’s Hadoop-Gremlin: advanced-topics/hadoop.md
    - Transaction Log: advanced-topics/transaction-log.md
    - JanusGraph Data model: advanced-topics/data-model.md
    - JanusGraph Bus: advanced-topics/janusgraph-bus.md
    - JanusGraph Cache: advanced-topics/cache.md
    - Technical Limitations: advanced-topics/technical-limitations.md
  - Common Questions: common-questions.md
  - Development: development.md
  - Changelog: changelog.md

What do you think?

We are able to redirect user the new content without any issue.

**Comment by li-boxuan** [comment]
Is it also possible to include https://github.com/JanusGraph/janusgraph/tree/master/janusgraph-examples in https://docs.janusgraph.org/? I think @pluradj has made a lot of efforts in that module but I don't see any link to that from our doc website.

**Reply by farodin91** [reply]
Sure we can do it. My plan with this pr was to restructure what we got.

**Comment by porunov** [comment]
@farodin91 do you know if this one was done? If so, are you good with closing this discussion?

**Reply by farodin91** [reply]
It should be done for now.
